device-types/Netgear/M4300-52G-PoEp.yaml

```diff
@@ -0,0 +1,341 @@
+    enabled: true
+    mgmt_only: false
+    poe_mode: pd
+    poe_type: type2-ieee802.3at
+  - name: 1/0/32
+    type: 1000base-t
+    enabled: true
+    mgmt_only: false
+    poe_mode: pd
+    poe_type: type2-ieee802.3at
+  - name: 1/0/33
+    type: 1000base-t
+    enabled: true
+    mgmt_only: false
+    poe_mode: pd
+    poe_type: type2-ieee802.3at
+  - name: 1/0/34
+    type: 1000base-t
+    enabled: true
+    mgmt_only: false
+    poe_mode: pd
+    poe_type: type2-ieee802.3at
+  - name: 1/0/35
+    type: 1000base-t
+    enabled: true
+    mgmt_only: false
+    poe_mode: pd
+    poe_type: type2-ieee802.3at
+  - name: 1/0/36
+    type: 1000base-t
+    enabled: true
+    mgmt_only: false
+    poe_mode: pd
+    poe_type: type2-ieee802.3at
+  - name: 1/0/37
+    type: 1000base-t
+    enabled: true
+    mgmt_only: false
+    poe_mode: pd
+    poe_type: type2-ieee802.3at
+  - name: 1/0/38
+    type: 1000base-t
+    enabled: true
+    mgmt_only: false
+    poe_mode: pd
+    poe_type: type2-ieee802.3at
+  - name: 1/0/39
+    type: 1000base-t
+    enabled: true
+    mgmt_only: false
+    poe_mode: pd
+    poe_type: type2-ieee802.3at
+  - name: 1/0/40
+    type: 1000base-t
+    enabled: true
+    mgmt_only: false
+    poe_mode: pd
+    poe_type: type2-ieee802.3at
+  - name: 1/0/41
+    type: 1000base-t
+    enabled: true
+    mgmt_only: false
+    poe_mode: pd
+    poe_type: type2-ieee802.3at
+  - name: 1/0/42
+    type: 1000base-t
+    enabled: true
+    mgmt_only: false
+    poe_mode: pd
+    poe_type: type2-ieee802.3at
+  - name: 1/0/43
+    type: 1000base-t
+    enabled: true
+    mgmt_only: false
+    poe_mode: pd
+    poe_type: type2-ieee802.3at
+  - name: 1/0/44
+    type: 1000base-t
+    enabled: true
+    mgmt_only: false
+    poe_mode: pd
+    poe_type: type2-ieee802.3at
+  - name: 1/0/45
+    type: 1000base-t
+    enabled: true
+    mgmt_only: false
+    poe_mode: pd
+    poe_type: type2-ieee802.3at
+  - name: 1/0/46
+    type: 1000base-t
+    enabled: true
+    mgmt_only: false
+    poe_mode: pd
+    poe_type: type2-ieee802.3at
+  - name: 1/0/47
+    type: 1000base-t
+    enabled: true
+    mgmt_only: false
+    poe_mode: pd
+    poe_type: type2-ieee802.3at
+  - name: 1/0/48
+    type: 1000base-t
+    enabled: true
+    mgmt_only: false
+    poe_mode: pd
+    poe_type: type2-ieee802.3at
+  - name: 1/0/49
+    type: 10gbase-t
+    enabled: true
+    mgmt_only: false
+    description: 10GBE / Stacking Ports
+  - name: 1/0/50
+    type: 10gbase-t
+    enabled: true
+    mgmt_only: false
+    description: 10GBE / Stacking Ports
+  - name: 1/0/51
+    type: 10gbase-x-sfpp
+    enabled: true
+    mgmt_only: false
+  - name: 1/0/52
+    type: 10gbase-x-sfpp
+    enabled: true
+    mgmt_only: false
+  - name: ServicePort
+    type: 1000base-t
+    enabled: true
+    mgmt_only: true
+    label: ServicePort
+    description: Ethernet Management Service Port
+  - name: vlan 1
+    type: virtual
+    enabled: true
+    mgmt_only: false
```

Comment:
This definition should be removed, we are really only focusing on physical definitions in the devicetype-library